images/scripts/fedora-33.bootstrap

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -eux
```

Comment:
It's safer to write this as `set -eux` on a separate line, as these scripts are often called through a shell. image-create executes it, so it's fine -- but it's a good habit anyway.